ios/FluentUI/HUD/HeadsUpDisplayTokenSet.swift

```diff
@@ -47,13 +36,22 @@ public class HeadsUpDisplayTokenSet: ControlTokenSet<HeadsUpDisplayTokenSet.Toke
                 }
 
             case .activityIndicatorColor:
+                let colorToken = theme.aliasTokens.colors[.foregroundLightStatic]
+                let lightColorValue = ColorValue(r: colorToken.light.r,
+                                                 g: colorToken.light.g,
+                                                 b: colorToken.light.b,
+                                                 a: 0.4)
```

Comment:
We talked about this some offline, but I think we need to follow up with design on this one. I don't think we should be making new colors by changing the alpha value of other tokens. I don't think that should block this change, but I think we should try to resist this becoming a new trend.

Reply:
If we rely on alpha/transparency, couldn't this be a problem if we have an unexpected color behind the control? If we don't explicitly set a fully opaque color, then we have no way to ensure we are actually fulfilling design goals (including accessibility ratios).

Reply:
Design confirmed that this is not to create a one-off usage token for the HUD component. This white token/opacity combination will likely be used to deal with future contrast issues. I would take this method over creating single use tokens especially if we're likely to run into similar contrast issues in the future.

Reply:
Personally I don't see this as any different than having a one-off token: the hardcoded alpha value is every bit as custom to this control as a hardcoded color would be. Let's follow up more with design, because I don't think this is setting a good precedent for dealing with creating grays from white.

Reply:
Separately, we shouldn't be hard-coding values like `0.4` directly in code like this. Magic numbers are bad!